src/docs/running/sourcing.md

```diff
@@ -3,22 +3,109 @@ title: "Sourcing"
 featured: ../images/featured/running.png
 ---
 
-## WORK IN PROGRESS NOTES
-
-- Organizing
-  - Until AuxEng is broadly known, start with in person conversation to set expectations
-    - Teams are signing up for change
-    - AuxEng won't be solving the hard problems in a silo
-    - Team must be interested in participating
-    - Check timelines
-  - Teams produce a 1-page proposal
-    - Interested teams produce a 1-page summary doc of project, timeline, technical and business impact goals
-- Selection
-  - Score on 3 criteria
-    - Business Impact
-    - Technical Impact
-    - Team fit
-  - AuxEng team individually scores proposal 0-10 on each criteria
-  - Discuss "Why" for each rating
-  - Participants get an opportunity to shift their votes
-  - Use your judgment, or be strictly mathematical. Either way you have the scoring data to measure
+After drumming up interest and [clarifying a team's mission](./expectations.md),
+managers and engineers should be ready to find ways to get teams interested.
+This might require a lot of outreach, or there may be tools important
+enough that many people will be delighted to speak with an Aux team. It's important
+to approach all these teams as similarly as possible, to give them all the best
+chance of driving value for an Aux Home team's goals in engagements.
+
+## Intro Meetings
+
+We regularly have teams approach us; looking for advice on building new
+services. We will explain to the team what our AuxEng program is and give
+them some general direction on how to proceed with their project.
+
+Most questions we get and most outline we give stem from the mission of the
+AuxEng team that sponsors the engineer. For example, we might have a team that
+heavily focuses on best practices for Python. We outline how we make life
+better for engineers that use our recommended tools, how we help setup automation
+and quality of life improvements for folks we engage with, and [generally pitch](./expectations.md#prepare-the-pitch)
```

Comment:
Broken link (I was able to get http://localhost:8000/running/expectations/#prepare-the-pitch to work locally, though)